docs/src/api/class-consolemessage.md

```diff
@@ -125,6 +125,12 @@ List of arguments passed to a `console` function call. See also [`event: Page.co
 
 URL of the resource followed by 0-based line and column numbers in the resource formatted as `URL:line:column`.
 
+## method: ConsoleMessage.page
+* since: v1.33
+- returns: <[Page]|[null]>
```

Comment:
how can Page be null if its not nullable on the server side?

Reply:
For future things like ServiceWorker.